libraries/apollo-normalized-cache-api-incubating/src/wasmJsMain/kotlin/com/apollographql/apollo3/cache/normalized/api/internal/ConcurrentMap.wasmJs.kt

```diff
@@ -0,0 +1,3 @@
+package com.apollographql.apollo3.cache.normalized.api.internal
+
+internal actual fun <K, V> ConcurrentMap(): MutableMap<K, V> = CommonConcurrentMap()
```

Comment:
No need to pay the wrapper price?

```suggestion
internal actual fun <K, V> ConcurrentMap(): MutableMap<K, V> = mutableMapOf()
```